percona-server-mongodb-4.4/Dockerfile.k8s

```diff
@@ -0,0 +1,140 @@
+FROM registry.access.redhat.com/ubi7/ubi-minimal
```

Comment:
```suggestion
FROM registry.access.redhat.com/ubi8/ubi-minimal
```

Reply:
Done for now,
Rebased on top of changes related to migration to centos8